changelog/19.0/19.0.0/summary.md

```diff
@@ -3,6 +3,7 @@
 ### Table of Contents
 
 - **[Major Changes](#major-changes)**
+  - **[Dropping Support for MySQL 5.7](#drop-support-mysql57)**
```

Comment:
Should I make this its own heading? Even above `Major Changes` or is it okay being the first thing in `Major Changes`?

Reply:
I see it's further down, so I guess that suffices?